pkg/console/subresource/configmap/configmap.go

```diff
@@ -18,23 +19,57 @@ const (
 	consoleConfigYamlFile   = "console-config.yaml"
 	clientSecretFilePath    = "/var/oauth-config/clientSecret"
 	oauthEndpointCAFilePath = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
-	// TODO: should this be configurable?  likely so.
-	documentationBaseURL = "https://docs.okd.io/4.0/"
-	brandingDefault      = "okd"
 	// serving info
 	certFilePath = "/var/serving-cert/tls.crt"
 	keyFilePath  = "/var/serving-cert/tls.key"
 )
 
-func DefaultConfigMap(cr *operatorv1.Console, rt *routev1.Route) *corev1.ConfigMap {
+// overridden by console config
+const (
+	defaultLogoutURL = ""
```

Comment:
dont we have a default URL for logout ?

Reply:
Nope, there should not be a default (empty string).  See console auth/logout handling:

https://github.com/openshift/console/blob/e267fcf4e177da5858c5c63d021d4f19f71646b8/frontend/public/module/auth.js#L56